fluent-bundle/src/bundle.ts

```diff
@@ -54,7 +55,7 @@ export class FluentBundle {
   constructor(
     locales: string | Array<string>,
     {
-      functions = {},
+      functions,
```

Comment:
This means that `functions` is now a required option? Or is this the `undefined` spread? I recall you mentioning that that doesn't work in typescript?

Also, does this have type information?

Reply:
That's a bit hard to read, but the types are defined in the lines below this one. It's due to how destructuring of parameters works in JS.

```ts
constructor(
    locales: string | Array<string>,
    {
      functions,
      useIsolating = true,
      transform = (v: string): string => v
    }: {
      functions?: Record<string, FluentFunction>;
      useIsolating?: boolean;
      transform?: TextTransform;
    } = {}
  ) { .. }
```

You can see that `function?` is possibly `undefined`. Both JS and TS have no problem with spreading it: `({...undefined})` evals to an empty object.

Reply:
Ah, thanks.